### PR TITLE
Add section to Contributing.md how to run tests 

### PR DESCRIPTION
**Pull request by begonaguereca**
Please feel free to update language or add any additional links! Another first stab at updating docs

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,6 +20,34 @@ Use the [Discussions](https://github.com/github/gh-gei/discussions) tab in this
 
 All work done by the maintainers of this repo is tracked in this repo using Issues. We have a hierarchical backlog with Epics at the top, broken down into Batches then broken down to Tasks (epic/batch/task is indicated via labels on the issues). You can see an example Epic and navigate down from there [here](https://github.com/github/gh-gei/issues/101).
 
+## Running specs 
+
+### In the terminal
+If you want to run tests selectively in the terminal, you can use dotnet test with `--filter` option.
+
+Here are some examples:
+1. Run all tests in `AdoApiTests` class. Navigate to `src/octoshiftcli.tests` and then execute the following command:
```
Comment (ArinGhazarian) [inline CONTRIBUTING.md:29]:
```suggestion
1. Run all tests in `AdoApiTests` class. Navigate to either `src` (where the `sln` file is) or `src/octoshiftcli.tests` (where the `csproj` file is) and then execute the following command:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,6 +20,34 @@ Use the [Discussions](https://github.com/github/gh-gei/discussions) tab in this
...
+ ```
+ dotnet test --filter AdoApiTests
+ ```
+
+2. Run a specific spec-`GetUserId_Should_Return_UserId` in `src/OctoshiftCLI.Tests/Octoshift/Services/AdoApiTests`. Navigate to `src/octoshiftcli.tests` and then execute the following command:
```
Comment (ArinGhazarian) [inline CONTRIBUTING.md:34]:
```suggestion
2. Run a specific spec-`GetUserId_Should_Return_UserId` in `src/OctoshiftCLI.Tests/Octoshift/Services/AdoApiTests`. Navigate to `src` or `src/octoshiftcli.tests` and then execute the following command:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -20,6 +20,34 @@ Use the [Discussions](https://github.com/github/gh-gei/discussions) tab in this
 
 All work done by the maintainers of this repo is tracked in this repo using Issues. We have a hierarchical backlog with Epics at the top, broken down into Batches then broken down to Tasks (epic/batch/task is indicated via labels on the issues). You can see an example Epic and navigate down from there [here](https://github.com/github/gh-gei/issues/101).
 
+## Running specs
```
Comment (ArinGhazarian) [inline CONTRIBUTING.md:23]: Something that I just noticed, these are not specs but rather tests :) 
```suggestion
## Running tests 
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -20,6 +20,34 @@ Use the [Discussions](https://github.com/github/gh-gei/discussions) tab in this
...
+### In the terminal
+If you want to run tests selectively in the terminal, you can use dotnet test with `--filter` option.
+
+Here are some examples:
+1. Run all tests in `AdoApiTests` class. Navigate to either `src` (where the `sln` file is) or `src/octoshiftcli.tests` (where the `csproj` file is) and then execute the following command:
+ ```
+ dotnet test --filter AdoApiTests
+ ```
+
+2. Run a specific spec-`GetUserId_Should_Return_UserId` in `src/OctoshiftCLI.Tests/Octoshift/Services/AdoApiTests`. Navigate to `src` or `src/octoshiftcli.tests` and then execute the following command:
```
Comment (ArinGhazarian) [inline CONTRIBUTING.md:34]:
```suggestion
2. Run a specific test-`GetUserId_Should_Return_UserId` in `src/OctoshiftCLI.Tests/Octoshift/Services/AdoApiTests`. Navigate to `src` or `src/octoshiftcli.tests` and then execute the following command:
```

**Review by ArinGhazarian — APPROVED** [review]

**Review by timrogers — APPROVED** [review]

**Pull request merged**